client/lib/analytics/ad-tracking/setup.js

```diff
@@ -209,3 +144,72 @@ function setupWpcomFloodlightGtag() {
 
 	window.gtag( 'config', TRACKING_IDS.wpcomFloodlightGtag );
 }
+
+export const setupAllTrackingScripts = () => {
```

Comment:
Given that this may be imported more than once, should we ensure that it runs only once per session?